packages/server/src/application/tests/mongo.spec.ts

```diff
@@ -1,47 +1,12 @@
-import mongoose from 'mongoose';
-import {
-  GenericContainer,
-  Network,
-  StartedNetwork,
-  StartedTestContainer,
-  Wait,
-} from 'testcontainers';
-import * as process from 'process';
 import application from '@application';
 
 describe('Sample Test', () => {
-  let network: StartedNetwork;
```

Comment:
Nice! I like solutions where code is being deleted. Looks like we've made bootstrapping much more easier.